### ISSUE TITLE
Add installation steps

### ISSUE THREAD

**Issue by imskr** [SwiftyChicken/dotfiles#2]
- please add installation steps

**Comment by imskr** [comment]
@SwiftyChicken

**Comment by jackddouglas** [comment]
@SwiftyChicken yes that would be very helpful.  I have read through all dotfiles, scripts, and the like but cannot get bspwm or anything else for that matter to work. It would be helpful for you to at least go over the thought process.  Thank you!

**Comment by godspeed1709** [comment]
@SwiftyChicken Most of the installation is achievable by reading the dotfiles but some things like the auto startup of polybar are not automatically working and thus would be better if you could provide a script or an installation guide. Thank you

**Comment by SwiftyChicken** [comment]
I'm currently cleaning up my dotfiles first. Ater that I will work on some sort of guide or wiki.
For the polybar issue I have explained here: #3
